languages/wikipron/src/remove_duplicates_and_split.sh

```diff
@@ -5,4 +5,5 @@ for TSV in ../tsv/*.tsv; do
     # Explicitly uses byte-wise comparison for sorting
     # rather than a locale-dependent comparison.
     LC_ALL=C sort -u -o "${TSV}" "${TSV}"
+    python generalized_split.py $TSV
```

Comment:
execute `chmod +x generalized_split.py` (if you haven't already), make sure it has the appropriate hashbang at the top, then just do `./generalized_split.py`.

Also do `"${TSV}"` instead of `$TSV`, which makes this robust to the presence of space or other escape characters in the filename path.